### PR TITLE
Add helper events for UIApplication Notifications

### PR DESCRIPTION
**Pull request by lordzsolt**
Based on #1895

### CODE REVIEW

**Comment by lordzsolt** [comment]
@gringoireDM I had to update the PR. Since the history of `develop` in the main repo was rewritten, git confused.

**Comment by freak4pc** [comment]
Can you rebase this? Sorry for the long time this took. Thanks!

**Comment by lordzsolt** [comment]
> Can you rebase this? Sorry for the long time this took. Thanks!

No worries. 🙂 

Rebased.

**Comment by lordzsolt** [comment]
Seems like the `ConcurrentDispatchQueueSchedulerTests.test_schedulePeriodic()` test is a bit flaky 🙁

**Comment by freak4pc** [comment]
It's flaky under Unix specifically, not too sure why right now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 
 ## [5.1.1](https://github.com/ReactiveX/RxSwift/releases/tag/5.1.1)
 
+* Add `ControlEvent` wrappers to `UIApplication` Notifications. #2116
```
Comment (freak4pc) [inline CHANGELOG.md:12]: This needs to be under unreleased, not 5.1.1

File: RxCocoa/iOS/UIApplication+Rx.swift [hunk 2]
```diff
@@ -8,17 +8,100 @@
...
 
-    extension Reactive where Base: UIApplication {
-        
-        /// Bindable sink for `networkActivityIndicatorVisible`.
-        public var isNetworkActivityIndicatorVisible: Binder<Bool> {
-            return Binder(self.base) { application, active in
-                application.isNetworkActivityIndicatorVisible = active
-            }
+extension Reactive where Base: UIApplication {
+
```
Comment (freak4pc) [inline RxCocoa/iOS/UIApplication+Rx.swift:15]:
```suggestion
```

File: RxCocoa/iOS/UIApplication+Rx.swift [hunk 3]
```diff
@@ -8,17 +8,100 @@
...
-    extension Reactive where Base: UIApplication {
-        
-        /// Bindable sink for `networkActivityIndicatorVisible`.
-        public var isNetworkActivityIndicatorVisible: Binder<Bool> {
-            return Binder(self.base) { application, active in
-                application.isNetworkActivityIndicatorVisible = active
-            }
+extension Reactive where Base: UIApplication {
+    
+    /// Bindable sink for `networkActivityIndicatorVisible`.
```
Comment (freak4pc) [inline RxCocoa/iOS/UIApplication+Rx.swift:16]:
```suggestion
    /// Bindable sink for `isNetworkActivityIndicatorVisible`.
```

**Comment by lordzsolt** [comment]
Glad to help 🙂

**Pull request merged**